lib/lib-auto-generated/src/main/resources/schema/org/sagebionetworks/repo/model/table/QueryResultBundle.json

```diff
@@ -39,9 +39,13 @@
 				"$ref": "org.sagebionetworks.repo.model.table.FacetColumnResult"
 			}
 		},
-		"sumFileSizes":{
-			"description":"The sum of the file size for all files in the given view query. Use mask = 0x40 to include in the bundle.",
+		"sumFileSizes": {
+			"description": "The sum of the file size for all files in the given view query. Use mask = 0x40 to include in the bundle.",
 			"$ref": "org.sagebionetworks.repo.model.table.SumFileSizes"
+		},
+		"estimatedUpToDate": {

```

Comment:
Maybe rethink this name, it's a tiny bit confusing. What does it mean if the up to date estimate is false? Maybe an indexStatus is enough? Or **isStale**?